packages/manager/apps/pci-instances/src/pages/instances/create/steps/region/StepTitle.component.tsx

```diff
@@ -30,12 +30,12 @@ export const StepTitle: FC<TStepTitleProps> = ({
 
   const regionStepTitle = useMemo(
     () =>
-      selectedRegion && !regionStepState?.isOpen
+      selectedRegion && !regionStep?.isOpen && regionStep?.isChecked
```

Comment:
I would have done that 
`const isRegionValidated = selectedRegion && regionStep?.isChecked && !regionStep?.isOpen`